### PR TITLE
Fix symbolic links to directories are not recognized as directories

### PR DESCRIPTION
**Pull request by plamentotev**
There is regression (#71) introduced with #33. The proposal is to revert #33 and make a new release of Plexus IO to fix the regression. Latter we can re-introduce the speed improvements, more carefully inspecting if they not change existing behavior. I've also added additional assertion to the existing tests that detects the regression.

Any objection?

cc @gnodet

### CODE REVIEW

File: src/main/java/org/codehaus/plexus/components/io/attributes/FileAttributes.java [hunk 1]
```diff
@@ -53,77 +52,62 @@
...
+            }
+            octalMode = (Integer) attrs.get( "mode" ) & 0xfff; // Mask off top bits for compatibilty. Maybe check if we
+                                                               // can skip this
+            symbolicLink = (Boolean) attrs.get( "isSymbolicLink" );
         }
         else
         {
-            names = "basic:*";
+            FileOwnerAttributeView fa = AttributeUtils.getFileOwnershipInfo( file );
+            this.userName = fa.getOwner().getName();
```
[the remark this answers is not in the document]
Comment (plamentotev) [reply src/main/java/org/codehaus/plexus/components/io/attributes/FileAttributes.java:102]: This just reverts old code, so I would rather keep it the way it was.

[the remark this answers is not in the document]
Comment (plamentotev) [reply src/main/java/org/codehaus/plexus/components/io/attributes/FileAttributes.java:102]: @sonatype-lift ignore

**Comment by plamentotev** [comment]
As there are no objections I'll proceed and revert the change and make a new release.

**Comment by gnodet** [comment]
> As there are no objections I'll proceed and revert the change and make a new release.

I'd rather fix the problem is possible.  Let me investigate today.

**Pull request merged**